src/main/app/src/app/zaken/zaak-view/zaak-view.component.ts

```diff
@@ -274,7 +274,7 @@ export class ZaakViewComponent extends ActionsViewComponent implements OnInit, A
             this.action = SideNavAction.MAIL_VERSTUREN;
         }, 'mail'));
 
-        if (!this.zaak.ontvangstbevestigingVerstuurd) {
+        if (!this.zaak.ontvangstbevestigingVerstuurd && this.zaak.status.naam !== 'Afgerond') {
```

Comment:
De naam van de laatste status van een zaak is configureerbaar in het ZTC. Het controleren van de naam kan dus niet. Er moet een boolean worden toegevoegd aan de zaak met naam ''isOpen". Deze kan in de backend gevuld worden vanuit net.atos.client.zgw.zrc.model.Zaak.isOpen()